packages/datadog-instrumentations/src/pg.js

```diff
@@ -37,13 +37,12 @@ function wrapQuery (query) {
       return retval
     }
 
-    const statement = pgQuery.text
     const callbackResource = new AsyncResource('bound-anonymous-fn')
     const asyncResource = new AsyncResource('bound-anonymous-fn')
     const processId = this.processID
 
     return asyncResource.runInAsyncScope(() => {
-      startCh.publish({ params: this.connectionParameters, statement, processId })
+      startCh.publish({ params: this.connectionParameters, pgQuery, processId })
```

Comment:
@bengl, did you have concerns about the name here? I remember you saying something about not wanting to tie the published message payload with the library variable name?

Reply:
It's a big redundant since it's published by `pg` in the first place. Just `query` might make more sense.

Reply:
> Just `query` might make more sense.

Reply:
👍

Reply:
@bengl and @rochdev `pgQuery` is actually coming from  `const pgQuery = queryQueue[queryQueue.length - 1] || activeQuery`

Reply:
would you like me to change both to query or keep the same?

Reply:
pgQuery has been updated to query in the params as well as in the pg plugin index